src/Adapter/PayloadSerializer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/*
+ * This file is part of the prooph/event-store.
+ * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ * 
+ * Date: 8/14/15 - 8:11 PM
+ */
+namespace Prooph\EventStore\Adapter;
+
+/**
+ * Interface PayloadSerializer
+ *
+ * @package Prooph\EventStore\Adapter
+ * @author Alexander Miertsch <alexander.miertsch.extern@sixt.com>
+ */
+interface PayloadSerializer 
+{
+    /**
+     * @param array $payload
+     * @return string
+     */
+    public function serializePayload(array $payload);
+
+    /**
+     * @param string $payloadStr
+     * @return array
+     */
+    public function unserializePayload($payloadStr);
+}
+ 
```

Comment:
hhhm, what exactly is missing? And how can I change it? Thought before you mean that I should add an empty line after the closing bracket but you mean something different, right?
My phpstorm is set up to use \n as line ending. What am I doing wrong?